platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -360,7 +360,7 @@ MGL_EXPORT
 #pragma mark - Managing Ambient Cache
 
 /**
- Sets the maximum ambient cache size in megabytes. The default maximum cache
+ Sets the maximum ambient cache size in bytes. The default maximum cache
```

Comment:
@jmkiley did you make a last-minute change to the method? I was under the impression that we had agreed setting the maximum ambient cache size in MB matched previous core and Android behavior so we were going to address it in a follow-up PR.

Reply:
I can change it to accept megabytes.

Reply:
The core method accepts bytes, as does the Android method. The issue, from what I understand, is that I made a typo (saying that the method accepts megabytes in one place, and bytes in another).

Reply:
ah, okay - I think the way it is now (saying `bytes`) is good then